activestorage/lib/active_storage/migration.rb

```diff
@@ -0,0 +1,27 @@
+class ActiveStorageCreateTables < ActiveRecord::Migration[5.1]
```

Comment:
It should probably have an extension that communicates it’s a template and excludes it from documentation, or we should otherwise exempt it if that’s at all possible.